classify-cluster-priority/README.md

```diff
@@ -0,0 +1,85 @@
+# Classifying clusters based on priority
+
+Contents:
+
+1. User stories
+2. Current practice
+3. Proposal
+4. Considerations and explanations
+5. Possible applications
+
+## User stories
+
+- As a cluster admin, I want to easily communicate the purpose and importance of a cluster to those interactig with it, so that they can act accordingly.
+- As Giant Swarm operations staff I want to understand the importance of clusters so I can make appropriate decisions, e. g. when prioritizing problem mitigation in an incident affecting multiple clusters.
```

Comment:
We might want to mention escalation as an important user story as well. We could derive from the label whether getting the customer or other engineers involved outside working hours is needed.

Reply:
Very much on point. From ours (honeybadger) point of view we even have some use cases, where we would like to route certain alerts on dev clusters directly to customers, without involving the team